### PR TITLE
feat(repo): update issue templates

### PR DESCRIPTION
**Pull request by 2manslkh**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: Bug report 🐛
 description: Report a bug
 labels: ["category.bug", "status.needs-triage"]
+title: "[BUG] "
```
Comment (d1onys1us) [inline .github/ISSUE_TEMPLATE/bug_report.yml:4]: can we improve this line a bit maybe as a call to action to succinctly describe the bug? similar to the project submission title

Comment (2manslkh) [reply .github/ISSUE_TEMPLATE/bug_report.yml:4]: maybe "Fix ___"?

Comment (d1onys1us) [reply .github/ISSUE_TEMPLATE/bug_report.yml:4]: do we really need the title you think? https://github.com/paradigmxyz/reth/blob/main/.github/ISSUE_TEMPLATE/bug.yml?plain=1#L1 <- i tried out the reth "create a bug" report and thought it was nice. just an empty field to clearly fill out the title.

Comment (2manslkh) [reply .github/ISSUE_TEMPLATE/bug_report.yml:4]: Makes sense, we have the category.bug tag already

**Review by dionysuzx — APPROVED** [review]

**Review by dantaik — APPROVED** [review]

**Comment by cenbeli** [comment]
Hi.how do i contribute

**Pull request merged**